frontend/mr-admin-flate/src/api/utkast/useAlleUtkast.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from "@tanstack/react-query";
+import { QueryKeys } from "../QueryKeys";
+import { mulighetsrommetClient } from "../clients";
+
+export function useAlleUtkast(avtaleId?: string) {
```

Comment:
På sikt må denne støtte å velge typen avtale eller tiltaksgjennomforing, men det tar jeg som en del av utkast for avtaler.